outline/electron/main.go

```diff
@@ -100,8 +99,26 @@ func main() {
 		os.Exit(oss.IllegalConfiguration)
 	}
 
+	prefix, err := url.PathUnescape(*args.proxyPrefix)
+	if err != nil {
+		log.Errorf("\"%s\" could not be URI-decoded", *args.proxyPrefix)
+		os.Exit(oss.IllegalConfiguration)
+	}
+	config := oss.Config{
```

Comment:
The alternative here would be to create a --config flag that takes a json object that is passed to Go. That way we wouldn't need to know all the fields here.